src/text/homepagebannercards.js

```diff
@@ -23,7 +23,7 @@ const HomepageBannerCards = ( page ) => {
 					</div>
 				</div>
 
-				<div className="row au-grids cards guideshomepagecards">
+				<div className="row cards guideshomepagecards">
```

Comment:
`guideshomepagecards` should probably be `cards--home` if we are trying to use BEM